packages/spanner/system-test/spanner.js

```diff
@@ -1105,7 +1088,9 @@ var spanner = new Spanner(env);
           });
 
           if (attempts < 2) {
```

Comment:
I wonder can it be that both these transactions end up using the same session, so the first one is actually failing due to that and not because of conflicts?